composer.json

```diff
@@ -12,16 +12,20 @@
             "app/AppCache.php"
         ]
     },
+    "autoload-dev": {
+        "files": [ "vendor/symfony/symfony/src/Symfony/Component/VarDumper/Resources/functions/dump.php" ]
+    },
     "require": {
+        "php": ">=7.0.22",
         "kunstmaan/bundles-cms": "~3.6.0",
-        "friendsofsymfony/http-cache-bundle": "~1.3.6",
-        "doctrine/doctrine-migrations-bundle": "~1.1.1",
-        "friendsofsymfony/user-bundle": "2.0.*@dev",
-        "kriswallsmith/spork": "~0.3.0",
-        "bureaublauwgeel/kunstmaanadminbundle": "3.6.0.7",
-        "bureaublauwgeel/routable-entity-bundle": "dev-develop",
-        "bureaublauwgeel/generatorbundle": "dev-develop",
-        "bureaublauwgeel/websitesettingsbundle": "dev-develop"
+        "friendsofsymfony/http-cache-bundle": "~1.3",
+        "doctrine/doctrine-migrations-bundle": "~1.3",
+        "friendsofsymfony/user-bundle": "2.0.0-beta1",
```

Comment:
Does this work with the current Kunstmaan version?

Reply:
Yes, in fact this is the exact version required in Kunstmaan 3.6.0.1. See: https://github.com/Kunstmaan/KunstmaanBundlesCMS/blob/3.6.1/composer.json#L31